exec/plugin-commands-script-runners/src/exec.ts

```diff
@@ -212,7 +212,12 @@ export async function handler (
           result[prefix].duration = getExecutionDuration(startTime)
         } catch (err: any) { // eslint-disable-line
           if (await isErrorCommandNotFound(params[0], err)) {
-            err.hint = buildCommandNotFoundHint(params[0], (await readProjectManifestOnly(opts.dir)).scripts)
+            err.message = `Command "${params[0]}" not found`
+
+            const nearestCommand = getNearestCommand(params[0], (await readProjectManifestOnly(opts.dir)).scripts)
+            if (nearestCommand) {
+              err.hint = `Did you mean "pnpm exec ${nearestCommand}"?`
```

Comment:
This doesn't make sense. The `pnpm exec` command is not for running scripts. `pnpm exec` is for running binaries from `node_modules/.bin` or just running any commands. For running scripts there is the `pnpm run` command.